autoload/sneak.vim

```diff
@@ -234,6 +226,20 @@ func! sneak#to(op, input, inputlen, count, register, repeatmotion, reverse, incl
     call sneak#util#nudge(a:reverse) "undo nudge for t
   endif
 
+  if is_op && 2 != a:inclusive && !a:reverse
+    " f edge case targeting EOL/EOB
+    if 1 == a:inclusive && virtcol('.') == virtcol('$') - 1
+      let s:save_virtualedit = &virtualedit
+      let &virtualedit = 'onemore'
+      augroup sneak_restore_virtualedit
+        autocmd!
+        autocmd CursorMoved * ++once let &virtualedit = s:save_virtualedit
+      augroup END
+    endif
+    " nudge right but do not wrap on EOL: go beyond EOL on virtual column if necessary
+    exec 'norm! l'
```

Comment:
This alters the `dvf` and `dvt` cases as discussed in #321 (but not towards the builtin `f`/`t` direction):

Pre-PR: `o_v` had *no effect* on these

Post-PR: includes 'one cell further' like:
- On `abcdef`, do `dvfd`, get `f`
- On `abcdef`, do `dvtd`, get `ef`